src/services/FileUploadService.ts

```diff
@@ -78,6 +83,28 @@ export class FileUploadService {
         return FileUploadModelResponse.fromModel(entry, this.baseUrl, humanReadable);
     }
 
+    public async expires(entry: FileUploadModel, expires: string): Promise<void> {
```

Comment:
maybe a better name? `calculateCustomExpration`

Reply:
Changed to more descriptive name